src/sentry/data_export/base.py

```diff
@@ -4,7 +4,7 @@
 from datetime import timedelta
 from enum import Enum
 
-SNUBA_MAX_RESULTS = 1000
+SNUBA_MAX_RESULTS = 50000
```

Comment:
Allow for more results per iteration (speeds up task by 50x). I believe the only limitation maybe memory issues by have a list that long with `dict`s in every spot. Not entirely sure on the memory limitations of workers in production so I think this is a good spot for now.

Reply:
Changed to 10k, since that's the actual max result limit. My bad!